netket/experimental/driver/tdvp.py

```diff
@@ -400,12 +427,6 @@ def info(self, depth=0):
         ]
         return "\n{}".format(" " * 3 * (depth + 1)).join([str(self)] + lines)
 
-    def _odefun(self, t, w, **kwargs):
```

Comment:
I did not mind leaving access to an internal function to see what the ode fun was doing...
Is it worth it removing it-?

Reply:
I could assign `self.odefun = outer_odefun` in `__init__`, if you want.